gui/viewer.py

```diff
@@ -24,12 +24,24 @@
 gi.require_version('Gtk', '3.0')
 from gi.repository import Gtk as gtk
 
+gi.require_version('Gdk', '3.0')
+from gi.repository import Gdk as gdk
```

Comment:
If you backmerge in the latest from `master` you will see I got rid of the renaming of `Gtk` to `gtk` (line 25 above). I don't see a good reason for this rename and it's basically an artifact from porting everything to Python 3. For consistency let's stick with `Gdk`, and then I think this PR will be good to go.

Reply:
I am sorry. My action was motivated by **line 25**. 
Back-merging **done** :+1: .